classes/PolicyPositions.php

```diff
@@ -115,12 +117,17 @@ private function getMemberPolicyPositions ($limit = NULL) {
 
                 // Make sure the dream actually exists
                 if (!empty($dream_info)) {
+                    $summary = '';
+                    if (array_key_exists($policy[0], $this->summaries)) {
+                      $summary = $this->divisions->generateSummary($this->summaries[$policy[0]]);
+                    }
                     $this->positions[] = array(
                         'policy_id' => $policy[0],
                         'policy' => $policy[1],
                         'desc' => $dream_info['full_sentence'],
                         'has_strong' => $dream_info['has_strong'],
-                        'position' => $dream_info['position']
+                        'position' => $dream_info['position'],
+                        'summary' => $summary
```

Comment:
Just as an aside, should this construct the array and then save it in both positions[] and positionsById (why does only the latter have score?). Or could we drop positions, looks like it's only used by topics and that could use positionsById easily enough (unless it's maintaining order in some way?).